docs/getting-started.md

```diff
@@ -50,20 +61,26 @@ $ cd docs/getting-started
 First of all, you need to define how the data will be organized (a.k.a database schema) in the application with Scalar DB database schema.
 Here is a database schema for the sample application. For the supported data types, please see [this doc](schema.md) for more details.
 
-```sql
-REPLICATION FACTOR 1;
-
-CREATE NAMESPACE emoney;
-
-CREATE TABLE emoney.account (
-    id TEXT PARTITIONKEY,
-    balance INT,
-);
+```json
+{
+  "emoney.account": {
+    "transaction": false,
+    "partition-key": [
+      "id"
+    ],
+    "clustering-key": [],
+    "columns": {
+      "id": "TEXT",
+      "balance": "INT"
+    },
+    "compaction-strategy": "STCS"
```

Comment:
I think this line can be removed since `compaction-strategy` is optional and the default value is "STCS".
The compaction strategy isn't important in this document.

Reply:
Thank you for your suggestion.
Document updated with the above suggestion.